salesagility/SuiteCRM
# FIXED #8134 -  Logo not in left-hand corner anymore

Label outdated by stating a position not used in the new template.

'CURRENT_LOGO_HELP' => 'This logo is displayed in the left-hand corner of the footer of the SuiteCRM application.', 
## Description




## Motivation and Context

https://github.com/salesagility/SuiteCRM/issues/8134
## How To Test This

1. go through install wizard
2. when at the configuration page go to branding
3. go to Current Logo: and hover over the `i` and check the new label has been changed
## Types of changes


### Final checklist


- [X] My code follows the code style of this project found [here](https://docs.suitecrm.com/cmmunity/contributing-code/coding-standards/).
